IdentityCore/src/MSIDError.h

```diff
@@ -241,7 +241,12 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
     
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->